scripts/build.ts

```diff
@@ -0,0 +1,76 @@
+import { pick } from '@hyperlane-xyz/utils';
+import fs from 'fs';
+import { parse } from 'yaml';
+import { CoreChains } from '../chains/core';
+
+function genJsExport(data, exportName) {
+  return `export const ${exportName} = ${JSON.stringify(data, null, 2)};`;
+}
+
+let chainMetadata = {};
+let chainAddresses = {};
+
+console.log('Parsing and copying chain data');
+for (const file of fs.readdirSync('./chains')) {
```

Comment:
what about the core.ts and schema files?

Reply:
core.ts gets copied over into index.ts which then gets compiled by ts.
The schema I forgot, I'll get that included too